### PR TITLE
DDF-4106 Migrates user component to react

### PR DESCRIPTION
**Pull request by andrewkfiedler**
#### What does this PR do?
Migrates the user component to react

#### Who is reviewing it? 
@vinamartin 
@djblue 
@samuelechu 
@AzGoalie 

#### What are the relevant tickets?
[DDF-4106](https://codice.atlassian.net/browse/DDF-4106)

#### Screenshots
screen shot 2018-09-05 at 5 37 15 pm

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/container/user/index.tsx [hunk 1]
```diff
@@ -10,6 +9,4 @@
  * <http://www.gnu.org/licenses/lgpl.html>.
  *
  **/
- --}}
-<div class="user-info-region">
-</div>
+export { default } from './user'
```
Comment (Bdthomson) [inline ui/packages/catalog-ui-search/src/main/webapp/react-component/container/user/index.tsx]: Really odd how github chooses to link old and new files together when all they share is a copyright comment.

**Comment by bellcc** [comment]
build now

**Comment by bellcc** [comment]
Beginning Hero

**Comment by bellcc** [comment]
Hero Successful  🎉 
- Logged in as admin
- Verified that user component looked as expected
- Verified that user component acted as expected

ddf-4106

**Comment by bellcc** [comment]
Extended Hero Successful 🎉 
- Logged in as guest
- Logged in with basic auth

**Pull request merged**